### PR TITLE
Image source changed and {% raw %} {% endraw %} added.

### PR DESCRIPTION
**Pull request by MS-Informatique**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

**Comment by fabaff** [comment]
Is that the successor of #1431?

File: source/_components/cover.garadget.markdown [hunk 1]
```diff
@@ -43,3 +43,50 @@ Configuration variables:
 
 
 If provided, the **access_token** will be used, otherwise the **username** and **password** will be used to automatically generate an access token at start time.
+
+
+**Example with more detail:**
+<p class='img'>
+<img src='/source/images/components/garadget/cover_garadget_details.png' />
```
Comment (fabaff) [inline source/_components/cover.garadget.markdown]: You need to add the image to the pull request.

Comment (MS-Informatique) [inline source/_components/cover.garadget.markdown]: The problem is that I have no permission to upload files or I am not doing it on the right way.

**Comment by MS-Informatique** [comment]
Yes it is the successor of #1431 
I am sure I am not doing the job on the right way but I don't know how to work with this GitHub.

File: source/_components/cover.garadget.markdown (hunk 1 above)
Comment (fabaff) [inline source/_components/cover.garadget.markdown]: `git add /source/images/components/garadget/cover_garadget_details.png` then commit and push to this branch.

Comment (MS-Informatique) [inline source/_components/cover.garadget.markdown]: Done :)
Thanks for your help!

**Pull request merged**